osu.Game/Screens/Play/BreakOverlay.cs

```diff
@@ -151,5 +157,30 @@ private void bindProcessor(ScoreProcessor processor)
             info.AccuracyDisplay.Current.BindTo(processor.Accuracy);
             info.GradeDisplay.Current.BindTo(processor.Rank);
         }
+
+        public bool OnPressed(GlobalAction action)
+        {
+            switch (action)
+            {
+                case GlobalAction.SkipCutscene:
+                    double startTime = -1;
+                    foreach (var b in breaks)
+                    {
+                        if(Time.Current > b.StartTime && Time.Current + skip_required_cutoff + fade_duration < b.EndTime)
```

Comment:
There should be a space between `if` and `(`.

You can probably write this whole thing without a loop, though.

`var activeBreak = breaks.FirstOrDefault(b => b.StartTime < Time.Current && b.EndTime > Time.Current + skip_required_cutoff);`

Reply:
The lead-in time should probably be half of the `skip_required_cutoff` to feel a bit more natural. Else you could potentially skip forward as little as 1ms.

Reply:
Yeah, without a loop is much better, I didn't think of that.

yeah skipping short, but noticeable amounts of time like half a second can be a bit jarring, so it makes sense that the lead in time would be less. I copy and pasted that line exactly from SkipOverlay.cs btw, so should it be changed there as well? (If so I would imagine in a different PR)